start.py

```diff
@@ -29,6 +30,10 @@ def main(args):
     if args.license_url and not args.license_credentials:
         raise Exception('--license-credentials is a required argument if --license-url is provided.')
 
+    if args.sdc_version and args.mapr_version.startswith('6.0.0') and not args.sdc_version.startswith('3'):
```

Comment:
I'd actually remove this check; there are too many permutations that we might have to support checking for if we go down this road.